src/js/components/picklist/picklist.scss

```diff
@@ -94,7 +94,7 @@
   position: relative;
 }
 
-.picklist__group:not(:last-child) {
+.picklist__group:not(:last-of-type) {
```

Comment:
I don't love this, but mark didn't add a border between the "Add" button and the list